### DISCUSSION TITLE
Is there a way for host to query presets in plug-in

### DISCUSSION THREAD

**Discussion by sagantech** [free-audio/clap#170] [Q&A] [answered]
At very least it would be nice to find out the name of the currently selected preset but also to change presets from the host would be useful. Is there anyway to do that?

**Comment by baconpaul** [comment, marked as answer]
there is currently no extension for this. we've talked about it but we shipped 1.0 without it.

This is a frequently requested extension, but as I'm sure you know, the other formats struggle here too so we want to think carefully before we do anything.

**Comment by baconpaul** [comment]
there is currently no extension for this. we've talked about it but we shipped 1.0 without it.

This is a frequently requested extension, but as I'm sure you know, the other formats struggle here too so we want to think carefully before we do anything.
